pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: layout.pug
+navigationTitle: Kafka
+title: Kafka
+menuWeight: 5
+excerpt: Deploying Kafka in a project
+---
+
+## Getting Started
+
+<!-- TODO: Fix ZooKeeper operator link - this relies on the zookeeper docs being available in order to pass pre-commit -->
+To get started with creating and managing a Kafka Cluster in a project, you first need to deploy the [Kafka operator](../../../../../workspaces/applications/catalog-applications/kafka-operator/) and the [ZooKeeper operator](../../../../../workspaces/applications/catalog-applications/) in the workspace where the project exists.
+
+Once you have the Kafka operator deployed, you can create Kafka Clusters by applying a `KafkaCluster` custom resource in a project's namespace. You can find various examples of the custom resources and their configurations in the [Kafka operator repository](https://github.com/banzaicloud/koperator/tree/master/config/samples).
```

Comment:
```suggestion
After you deploy the Kafka operator deployed, create Kafka Clusters by applying a `KafkaCluster` custom resource in a project's namespace. Refer to the the [Kafka operator repository](https://github.com/banzaicloud/koperator/tree/master/config/samples) for examples of the custom resources and their configurations.
```

Reply:
fixed in 2fcad2ad6824e86564597d3ab4a8c318b7102fda